Formula/samba.rb

```diff
@@ -23,12 +23,19 @@ class Samba < Formula
     sha256 x86_64_linux:   "f97c1ddf6a5dde556437655a11003015aec5acaa74a55bf702e68732d625454f"
   end
 
+  depends_on "cmocka" => :build
+  depends_on "pkg-config" => :build
   # configure requires python3 binary to be present, even when --disable-python is set.
   depends_on "python@3.11" => :build
   depends_on "gnutls"
+  # icu4c can get linked if detected by pkg-config and there isn't a way to force disable
+  # without disabling spotlight support. So we just enable the feature for all systems.
+  depends_on "icu4c"
```

Comment:
Should avoid us deprioritizing broken Linux bottle during `icu4c` PRs as failure should be visible with macOS too.

As far as I could tell, there wasn't a way to just disable `icu4c` support.

Dependency is used in most other repositories I checked, e.g. MacPorts, FreeBSD, Arch, Debian, Fedora, Gentoo, ...